### PR TITLE
[OT185-21]Activity model and Repo

### PR DESCRIPTION
**Pull request by joelrodriguez-rgb**
## Summary
https://alkemy-labs.atlassian.net/browse/OT185-21
COMO desarrollador 
QUIERO agregar la entidad Activity
PARA representar en la implementación la estructura de datos

Criterios de aceptación: 
Nombre de tabla: activity. Campos:
name: VARCHAR NOT NULL
content: TEXT NOT NULL
image: VARCHAR NOT NULL
timestamps y softDelete

### Type of change
- [x] New feature
### Checklist

- [x] Traceability between this change and Jira.
- [x] ```mvn clean install``` was run and all tests completed successfully.
- [x] There are no unused variables and/or imports in the modified classes.
- [x] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [x] Slf4j was used for the application log instead ```System.out```.
- [x] Public methods are documented with ```javadoc```.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/domain/model/Activity.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import java.util.Objects;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@Entity
+@Table(name = "activity")
+@Where(clause = "is_active=true")
+@SQLDelete(sql = "UPDATE alkymer SET is_active=false WHERE alkymer_id=?")
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/model/Activity.java]: Corregir la query: `UPDATE activity SET is_active=false WHERE activity_id =?`

File: src/main/java/com/alkemy/ong/domain/model/Activity.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+@EntityListeners(AuditListener.class)
+public class Activity implements Auditable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "activity_id")
+    Long id;
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/model/Activity.java]: Por favor, hacer los atributos de clase privados.

**Review by manerajona — CHANGES_REQUESTED** [review]

File: src/main/java/com/alkemy/ong/domain/model/Activity.java (hunk 1 above)
Comment (joelrodriguez-rgb) [reply src/main/java/com/alkemy/ong/domain/model/Activity.java]: Resuelto.

File: src/main/java/com/alkemy/ong/domain/model/Activity.java (hunk 2 above)
Comment (joelrodriguez-rgb) [reply src/main/java/com/alkemy/ong/domain/model/Activity.java]: Resuelto.

**Review by manerajona — APPROVED** [review]

**Pull request merged**